code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -1104,20 +1104,27 @@
 
 /obj/item/clothing/head/helmet/space/hardsuit/powerarmor_advanced
 	name = "Advanced power helmet"
-	desc = "It's an advanced power armor Mk I helmet. It looks somewhat threatening."
+	desc = "It's an advanced power armor X-01 helmet. It looks somewhat threatening."
 	icon_state = "hardsuit0-advpa1"
 	item_state = "advpa1_helm"
 	hardsuit_type = "advpa1"
-	armor = list(MELEE = 50, BULLET = 48, LASER = 25, ENERGY = 25, BOMB = 48, BIO = 0, RAD = 0, FIRE = 0, ACID = 25)
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/resonator, /obj/item/mining_scanner, /obj/item/t_scanner/adv_mining_scanner, /obj/item/gun/energy/kinetic_accelerator, /obj/item/twohanded/required/kinetic_crusher)
```

Comment:
>/obj/item/twohanded/required/kinetic_crusher

Hubba hubba 😍

Honestly, I would be fine with this having the same exact armor as an explorer suit with the armor plating component if this includes kinetic crusher in the suit slot.

Would carve out a nice niche for it instead of it being somewhere in the list of explorer suit -> ash drake armor/inquisitors hardsuit -> HECK suit.

Reply:
you sure about the fire armor being 0? I'd feel like for being necropolis chest loot, most miners would just find the nearest lava pool and proceed to yeet it in-

However, I am no miner, and you frequently are so

Reply:
testing this change out to make sure it works right

Reply:
well it works, but now the question I have is

what if it was not Goliath plates, but some other thematically fitting item?

Reply:
This is actually pointing out what you changed, so the red line is removing :)

Reply:
Should be resolved by the newest commit